cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/ThreeWindingsTransformerConversion.java

```diff
@@ -20,6 +20,7 @@
 /**
  * @author Luma Zamarreño <zamarrenolm at aia.es>
  */
+@Deprecated
```

Comment:
Not necessary to deprecate the methods if the class is deprecated.
Also comment:
```java
/**
 * @deprecated Use {@link NewThreeWindingsTransformerConversion} instead.
 */
@Deprecated
public class ThreeWindingsTransformerConversion extends AbstractConductingEquipmentConversion {
...
```

Reply:
Done